jsonminify/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for jsonminify 0.4.1
+// Project: https://github.com/fkei/JSON.minify
+// Definitions by: Dan Homola <https://github.com/no23reason>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface MinifyFn {
+    (json: string): string;
+}
+
+declare var minify: MinifyFn;
+
+declare module "jsonminify" {
```

Comment:
could you strip `declare module "jsonminify" {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred